src/cart/crt_swim.c

```diff
@@ -168,9 +215,17 @@ static void crt_swim_srv_cb(crt_rpc_t *rpc_req)
 	else if (snd_delay > max_delay)
 		swim_net_glitch_update(ctx, from_id, snd_delay - max_delay);
 
-	rc = swim_parse_message(ctx, from_id,
-				rpc_swim_input->upds.ca_arrays,
-				rpc_swim_input->upds.ca_count);
+	if (CRT_SWIM_SHOULD_FAIL(d_fa_swim_drop_rpc, self_id)) {
+		rc = -d_fa_swim_drop_rpc->fa_err_code;
+		D_ERROR("*** DROP incoming opc %#x with %zu updates "
```

Comment:
Probably a warning, and you don't need the "*** " prefix.

Reply:
I definitely would like to see this message even if user specify ERR debug level. I put '***' before all messages related to fault injection. So, you can easily find them out.